oauth2server/proj/settings/cloud_foundry_settings.py

```diff
@@ -76,22 +72,6 @@
     DB_USERNAME = credentials.get('username', '')
```

Comment:
If we default the DB credentials to empty strings, could this cause a silent failure? Would it be worth failing (with Splunk output) if the keys are not found?

Reply:
Removed empty string. If 'vcap_services' is set and there are no DB settings the micro service will fail with type errors.